distribution/lib/Standard/Visualization/0.0.0-dev/src/Helpers.enso

```diff
@@ -136,9 +138,18 @@ Vector.default_visualization self = Id.table
    Transform the vector into text for displaying as part of its default
    visualization.
 Vector.to_default_visualization_data : Text
-Vector.to_default_visualization_data self =
-    json = self.take 100 . to_json
-    json.to_text
+Vector.to_default_visualization_data self = render_vector self
+
+render_vector object depth=0 =
+    case object of
+        _ : Vector -> if depth == 5 then "[...]" else
```

Comment:
In `Json` we parametrize with `max_depth`. Here we just use a constant `5` instead.
I think we should parametrize in both places, no?